_people/Michael_Pajkos.markdown

```diff
@@ -0,0 +1,26 @@
+---
+name: Michael Pajkos 
+lastname: Pajkos
+group: postdocs
+position: Postdoctoral Researcher
+address: Cahill Center for Astronomy and Astrophysics
+institution: California Institute of Technology
+email: mpajkos@caltech.edu
+website: [Personal Website](https://sites.google.com/msu.edu/michaelpajkos/home) 
```

Comment:
This markdown would need to be quoted, but the yaml content is actually not parsed as markdown (this line actually leads to a yaml parse error for me, but I have no idea if it's actually illegal, because yaml is a foresaken language).
```suggestion
website: https://sites.google.com/msu.edu/michaelpajkos/home
```

Reply:
Ping @pajkosmi to fix the yaml?